File: src/com/github/unchama/seichiassist/MineBlock.java
package com.github.unchama.seichiassist;

public class MineBlock{
	public int after;
	public int before;
	public int increase;

	MineBlock(){
		after = 0;
		before = 0;
		increase = 0;
	}
	public void setIncrease(){
		increase = after - before;
	}
}

File: src/main/scala/com/github/unchama/seichiassist/data/player/MineBlock.scala
package com.github.unchama.seichiassist.data.player

// FIXME semantics is not clear
class MineBlock() {
  var after: Long = 0
  var before: Long = 0
  var increase: Long = 0

  def setIncrease(): Unit = {
    increase = after - before
  }
}
